samples/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -21,8 +21,8 @@
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_SCANNING_LOG_LEVEL);
 
 
-#define DEVICE_NAME	CONFIG_DESKTOP_BLE_SCANNING_DEVICE_NAME
-#define DEVICE_NAME_LEN	(sizeof(DEVICE_NAME) - 1)
+#define DEVICE_NAME		CONFIG_DESKTOP_BLE_SCANNING_DEVICE_NAME
+#define DEVICE_NAME_LEN		CONFIG_DESKTOP_BLE_SHORT_NAME_LEN
```

Comment:
We can get this len based on `sizeof(DEVICE_NAME)-1`

Reply:
and use short scanning device name, also making it visible to advertiser module

Reply:
Sorry, the complete name will be put in scan response data. In `ad` it will always be shortened name.